slave-base/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM openshift/origin
+FROM quay.io/openshift/origin-cli:v3.11@sha256:25d6e5bc4d7f5521effb20a8b293b57d16224ed4a3c37d3a701c1f559359333e
```

Comment:
we can move this to origin-cli but we don't want to pin to a version like this.

also the rhel image should be moved at the same time.

(as should the master image)

We already did this in the master branch for v4.

Reply:
I have removed the digest from the image but I'm not sure what to use for the rhel Dockerfile. If you send me an image uri I can include it in this PR

Reply:
should it be `registry.access.redhat.com/openshift3/ose-cli:v3.11`?

Reply:
yes